lib/tessel/deploy.js

```diff
@@ -334,13 +334,14 @@ actions.tarBundle = function(opts) {
 
           return rules.concat(patterns);
         }, []).map(function(rule) {
-
+          // Ensure it uses proper paths for OS
+          rule = path.normalize(rule);
           if (rule[rule.length - 1] === '/') {
-            rule += '**/*.*';
+            rule += path.normalize('**/*.*');
           }
 
           if (rule[rule.length - 1] !== '*' && rule.indexOf('.') === -1) {
-            rule += '/**/*.*';
+            rule += path.normalize('/**/*.*');
           }
 
```

Comment:
I'm not sure it does - this is the error AppVeyer was giving me: https://ci.appveyor.com/project/rwaldron/t2-cli/build/25/job/kgqcn0c4eo0ml4uj